packages/protocol/contracts/bridge/TokenVault.sol

```diff
@@ -168,12 +167,10 @@ contract TokenVault is EssentialContract {
         message.to = to;
         message.gasLimit = gasLimit;
         message.processingFee = processingFee;
-        message.depositValue = msg.value - processingFee;
+        message.callValue = msg.value - processingFee;
         message.refundAddress = refundAddress;
         message.memo = memo;
-
-        // prevent future PRs from changing the callValue when it must be zero
-        if (message.callValue != 0) revert TOKENVAULT_INVALID_CALL_VALUE();
```

Comment:
I think I added this comment to prevent both `depositValue` and `callVaue` to be set to `msg.value - processingFee` to cause a double-deposit.